test/functional/Valhalla/src/org/openj9/test/lworld/NullRestrictedTypeOptTests.java

```diff
@@ -343,4 +343,43 @@ static private void evalTestEA8(int i) {
 			escape8 = p;
 		}
 	}
+
+   public static class TestStoreToNullRestrictedField {
+      public PrimPair nullRestrictedInstanceField;
+      public static PrimPair nullRestrictedStaticField;
+
+      public void replaceInstanceField(Object val) {
+         nullRestrictedInstanceField = (PrimPair) val;
+      }
+
+      public static void replaceStaticField(Object val) {
+         nullRestrictedStaticField = (PrimPair) val;
+      }
+   }
+
+   public static primitive class TestWithFieldStoreToNullRestrictedField {
+      public PrimPair nullRestrictedInstanceField;
+
+      public TestWithFieldStoreToNullRestrictedField(Object val) {
+         this.nullRestrictedInstanceField = (PrimPair) val;
+      }
+   }
+
+   @Test(expectedExceptions = java.lang.NullPointerException.class)
+   static public void testStoreNullValueToNullRestrictedInstanceField() throws Throwable {
+      new TestStoreToNullRestrictedField().replaceInstanceField(new PrimPair(1, 2));
+      new TestStoreToNullRestrictedField().replaceInstanceField(null);
+   }
+
+   @Test(expectedExceptions = java.lang.NullPointerException.class)
+   static public void testStoreNullValueToNullRestrictedStaticField() throws Throwable {
+      TestStoreToNullRestrictedField.replaceStaticField(new PrimPair(1, 2));
+      TestStoreToNullRestrictedField.replaceStaticField(null);
+   }
+
+   @Test(expectedExceptions = java.lang.NullPointerException.class)
+   static public void testWithFieldStoreToNullRestrictedField() throws Throwable {
+      new TestWithFieldStoreToNullRestrictedField(new PrimPair(1, 2));
+      new TestWithFieldStoreToNullRestrictedField(null);
+   }
```

Comment:
For whatever reason, the JVM team uses tabs for indentation rather than spaces.  I think that generally applies to tests as well.  May I ask you to use tabs here to make the indentation consistent with the rest of the file?

Reply:
Addressed in 7725103